src/error.rs

```diff
@@ -135,6 +135,17 @@ impl ResolveError {
             column: error.column(),
         })
     }
+
+    #[cold]
+    #[must_use]
+    pub fn from_simd_json_error(path: PathBuf, error: &simd_json::Error) -> Self {
+        Self::Json(JSONError {
+            path,
+            message: error.to_string(),
+            line: 0, // simd_json doesn't provide line/column info
```

Comment:
For the error exception, could we parse it again with serde_json to get the precise error line/column information, so the user experience remains unchanged?